### DISCUSSION TITLE
Adding modifiers for x-text ?

### DISCUSSION THREAD

**Discussion by Chandler-Zhu** [alpinejs/alpine#1289] [3. Feature ideas]
I hope in the next release there will be modifiers for `x-text`, such as `x-text.toLocaleString('en-US')`
pseudo code:
```
<div x-data="{subtotal:1645.98}">
      <h2>
        Your subtotal  is <span x-text.toLocaleString('en-US')="subtotal || '0.00'"></span> $
      </h2>
</div>
```

result: Your subtotal is 1,645.98 $

**Comment by SimoTod** [comment]
There are already a few ways to do it now. For example

```html
<script>
    function formatMoney(value) {
        return value.toLocaleString('en-US', { style: 'currency', currency: 'USD' })
    }
</script>
<div x-data="{subtotal:1645.98}">
      <h2>
        Your subtotal  is <span x-text="formatMoney(subtotal || '0.00')"></span> $
      </h2>
</div>
```

**Reply by Chandler-Zhu** [reply]
that's an elegant solution, I was just thinking adding commas in large numbers is so common these days, maybe it can be part of this library, something like `x-text.formatNum`
